lib/services/usb-livesync-service.ts

```diff
@@ -10,7 +10,8 @@ import Future = require("fibers/future");
 
 export class UsbLiveSyncService extends usbLivesyncServiceBaseLib.UsbLiveSyncServiceBase implements IUsbLiveSyncService {
 	private excludedProjectDirsAndFiles = [
-		"app_resources"
+		"**/app_resources/**",
```

Comment:
i think it is better to remove this rule entirely. We should let the users to sync their resources.